treadmill/appmgr/configure.py

```diff
@@ -146,7 +146,7 @@ def configure(tm_env, event):
 
     # Generate the supervisor's run script
     app_run_cmd = ' '.join([
-        os.path.join(treadmill.TREADMILL, 'bin', 'treadmill'),
+        os.path.join('/home', 'centos', 'treadmill', 'env', 'bin', 'treadmill'),
```

Comment:
This should not be changed; `treadmill.TREADMILL` will be defined to where ever the `treadmill` script resides, this is not the right thing to do.